frontend/src/app/features/dashboard/dashboard/dashboard.component.html

```diff
@@ -33,10 +33,25 @@
     </button>
     <div [ngSwitch]="sidebarType$ | async">
         <so-sidebar class="sidebar" *ngSwitchCase="SidebarType.Create" (closed)="closeSidebar()">
-            <so-task-form></so-task-form>
+            <so-task-form 
+                    (cancel)="closeSidebar()"
+                    (submitted)="createTask($event)"
+                    [initialValue]="null"
```

Comment:
What is the point to send null? Have it as default value or do not pass at all

Reply:
Fixed